### ISSUE TITLE
Get Field Component via refs

### ISSUE THREAD

**Issue by erickjth** [final-form/react-final-form#141]
### Are you submitting a **bug report** or a **feature request**?
feature request

### What is the current behavior?
There is no a way to get Field components via refs.

### What is the expected behavior?
Access to the rendered component in a Field `<Field {....} component={MyCustomField} />`.

### What's your environment?
- final-form": "^4.2.0"
- react-final-form": "^3.1.0

### Other information
I am using this library in a React Native App, so the behavior that I would like to implement is this: Focus a field when clicking on "Next" key in the virtual keyboard on iOS.

**Comment by bogdansoare** [comment]
also having the same issue

**Comment by ibratoev** [comment]
You can use a simple func to render your component like:
```
renderMyCustomField = props => <MyCustomField {...props} ref={customFieldRef] />
```

**Comment by erickjth** [comment]
Would be nice to have the official **Forwarding Refs** API implemented in each components to pass down the reference.

**Comment by kopax** [comment]
I am using react-native and react-final-form and this is kind of deal breaker not to have the forward ref, because I can't solve the UX requirement as follow: https://stackoverflow.com/questions/32748718/react-native-how-to-select-the-next-textinput-after-pressing-the-next-keyboar

Any update on this ?

**Comment by kopax** [comment]
it's been over 2 years that this issue has been reported and that didn't get any attention, I have opened https://github.com/final-form/react-final-form/pull/781 and offer to [added a warning](https://github.com/final-form/react-final-form/pull/781/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8) on top of the [`README.md`](https://github.com/final-form/react-final-form/pull/781/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8) to prevent people wasting time if they required that feature.

This would save a lot of time to others, by not reading nor implementing with that library. We will have to replace all of it, I'll personally recommend using [`react-hook-form`](https://react-hook-form.com/) to implement form in react, which is not an abandoned library.

**Comment by kopax** [comment]
Hi and thanks @erikras for taking care of it, this could remain open till we can test it on our end don't you think?

**Comment by erikras** [comment]
Published fix in [`v6.5.0`](https://github.com/final-form/react-final-form/releases/tag/v6.5.0).
